examples/hello_world/hello_world.robot

```diff
@@ -0,0 +1,32 @@
+*** Settings ***
+Suite Setup                     Setup
+Suite Teardown                  Teardown
+Test Setup                      Reset Emulation
+Test Teardown                   Test Teardown
+Resource                        ${RENODEKEYWORDS}
+
+*** Variables ***
+${UART}                         sysbus.uart0
+
+*** Keywords ***
+Create Machine
+    Execute Command             mach create
+    Execute Command             machine LoadPlatformDescription @platforms/boards/arduino_nano_33_ble.repl
+
+    Execute Command             sysbus LoadELF @${CURDIR}/binaries/hello_world/hello_world.ino.elf
```

Comment:
please verify if the path is ok after you drop it from the repo

Reply:
It passes tests so it's ok